docs/auto-generating-selectors.md

```diff
@@ -0,0 +1,69 @@
+# Auto Generating Selectors
+
+It is recommended to use selectors when using either the properties or actions from the store.
+
+```javascript
+const bears = useBearStore((state) => state.bears)
+```
+
+However, writing these could be tedious, but you can auto-generate them
+
+## create the following function: `createSelectors`
+
+```typescript
+import create, { StateCreator, State, StoreApi, UseStore } from 'zustand'
```

Comment:
`UseStore` is deprecated.
I think you want to create a codesandbox example and add the link in this doc.

Reply:
You are right, I should have tested that. I created a CodeSandbox and update the examples:
https://codesandbox.io/s/zustand-auto-generate-selectors-9i0ob3?file=/src/App.tsx